components/utils/PopulationMultiSelect.tsx

```diff
@@ -38,15 +38,22 @@ export default function PopulationMultiSelect() {
       if (value.length > 0 && !value.includes(Enums.PopulationType.IPP)) {
         newDesiredPopulations.push(Enums.PopulationType.IPP);
       }
-      // add denominator if numerator, numerator exclusion, or denominator exception are selected
+      // add denominator if numerator, numerator exclusion, denominator exclusion, or denominator exception are selected
       if (
-        [Enums.PopulationType.NUMER, Enums.PopulationType.NUMEX, Enums.PopulationType.DENEXCEP].some(p =>
-          value.includes(p)
-        ) &&
+        [
+          Enums.PopulationType.NUMER,
+          Enums.PopulationType.NUMEX,
+          Enums.PopulationType.DENEX,
+          Enums.PopulationType.DENEXCEP
+        ].some(p => value.includes(p)) &&
         !value.includes(Enums.PopulationType.DENOM)
       ) {
         newDesiredPopulations.push(Enums.PopulationType.DENOM);
       }
+      // add numerator if numerator exclusion is selected
+      if ([Enums.PopulationType.NUMEX].some(p => value.includes(p)) && !value.includes(Enums.PopulationType.NUMER)) {
```

Comment:
```suggestion
      if (value.includes(Enums.PopulationType.NUMEX) && !value.includes(Enums.PopulationType.NUMER)) {
```
slight simplification